plugins/vst_base/RemoteVstPlugin.cpp

```diff
@@ -40,10 +40,6 @@
 
 #ifdef LMMS_BUILD_LINUX
 
-#ifndef NOMINMAX
-#define NOMINMAX
-#endif
-
```

Comment:
Please keep this or modify relevant CMake files, we don't propagate all C++ flags when building `RemoteVstPlugin` using `ExternalProject`.